src/managedwriter/stream_connection.ts

```diff
@@ -183,15 +190,23 @@ export class StreamConnection extends EventEmitter {
   }
 
   private handleData = (response: AppendRowsResponse) => {
-    this.trace('data arrived', response);
-    const pw = this.getNextPendingWrite();
-    if (!pw) {
+    this.trace('data arrived', response, this._pendingWrites.length);
```

Comment:
This is probably too much logging, it will log one line per request. If it is debug logging then it is fine.

Reply:
those `trace` calls are only on DEBUG mode.

Reply:
Is there a LOG mode? I think those reconnect errors need to be in LOG mode. Even for debug mode, per request logging might be too much, but I am less concerned.

Reply:
gonna be the first user of that new logger @feywind 🚀